kld/src/api/spec.yaml

```diff
@@ -159,7 +225,7 @@ paths:
                 description: Funding
               state:
                 type: string
-                enum: [usable, ready, pending]
+                enum: [CHANNELD_NORMAL, OPENINGD]
```

Comment:
Which app will use this?  Some states do not implement it now.  Does the app work well?

- `CHANNELD_AWAITING_LOCKIN`,
- `CHANNELD_SHUTTING_DOWN`,
- `CLOSINGD_SIGEXCHANGE`,
- `CLOSINGD_COMPLETE`,
- `AWAITING_UNILATERAL`,
- `FUNDING_SPEND_SEEN`,
- `ONCHAIN`,
- `DUALOPEND_OPEN_INIT`,
- `DUALOPEND_AWAITING_LOCKIN`,
- `CHANNELD_AWAITING_SPLICE`

Reply:
[RTL](https://github.com/Ride-The-Lightning/RTL)
I've been testing it which has resulted in these fixes. Yeah it works a lot better with two correct states than none. Will add more as needed.